<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QWidget" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>689</width>
    <height>490</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>689</width>
    <height>490</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>689</width>
    <height>490</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>BirdFlock</string>
  </property>
  <widget class="QPushButton" name="ResetButton">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>80</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Reset</string>
   </property>
  </widget>
  <widget class="Line" name="line">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>60</y>
     <width>661</width>
     <height>20</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>0</y>
     <width>371</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Bird Flock Simulator V2.0 (Final Version)</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_15">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>30</y>
     <width>191</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>9</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Made by Max Elliott - ID 1434717</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Pause_Button">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>80</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Pause</string>
   </property>
  </widget>
  <widget class="QTabWidget" name="tabWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>110</y>
     <width>671</width>
     <height>371</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <widget class="QWidget" name="Flock_Tab">
    <attribute name="title">
     <string>Flock Objects</string>
    </attribute>
    <widget class="QGroupBox" name="Green_Group">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>321</width>
       <height>151</height>
      </rect>
     </property>
     <property name="title">
      <string>Green birds</string>
     </property>
     <widget class="QLabel" name="label_6">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>80</y>
        <width>101</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Separation radius</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_7">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>20</y>
        <width>47</width>
        <height>13</height>
       </rect>
      </property>
      <property name="text">
       <string>No. birds</string>
      </property>
     </widget>
     <widget class="QSlider" name="G_Det_Slider">
      <property name="geometry">
       <rect>
        <x>110</x>
        <y>110</y>
        <width>160</width>
        <height>22</height>
       </rect>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
     <widget class="QLabel" name="label_8">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>50</y>
        <width>47</width>
        <height>13</height>
       </rect>
      </property>
      <property name="text">
       <string>Speed</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_9">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>110</y>
        <width>101</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Detection radius</string>
      </property>
     </widget>
     <widget class="QSpinBox" name="G_Count_Box">
      <property name="geometry">
       <rect>
        <x>110</x>
        <y>20</y>
        <width>51</width>
        <height>22</height>
       </rect>
      </property>
      <property name="maximum">
       <number>900</number>
      </property>
     </widget>
     <widget class="QSlider" name="G_Sep_Slider">
      <property name="geometry">
       <rect>
        <x>110</x>
        <y>80</y>
        <width>160</width>
        <height>22</height>
       </rect>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
     <widget class="QSlider" name="G_Speed_Slider">
      <property name="geometry">
       <rect>
        <x>110</x>
        <y>50</y>
        <width>160</width>
        <height>22</height>
       </rect>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
     <widget class="QLabel" name="G_Speed_Value">
      <property name="geometry">
       <rect>
        <x>280</x>
        <y>50</y>
        <width>31</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>0</string>
      </property>
     </widget>
     <widget class="QLabel" name="G_Sep_Value">
      <property name="geometry">
       <rect>
        <x>280</x>
        <y>80</y>
        <width>31</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>0</string>
      </property>
     </widget>
     <widget class="QLabel" name="G_Det_Value">
      <property name="geometry">
       <rect>
        <x>280</x>
        <y>110</y>
        <width>31</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>0</string>
      </property>
     </widget>
    </widget>
    <widget class="QGroupBox" name="Red_Group">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>160</y>
       <width>321</width>
       <height>171</height>
      </rect>
     </property>
     <property name="title">
      <string>Predators</string>
     </property>
     <widget class="QLabel" name="label_10">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>80</y>
        <width>101</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Separation radius</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_11">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>20</y>
        <width>47</width>
        <height>13</height>
       </rect>
      </property>
      <property name="text">
       <string>No. birds</string>
      </property>
     </widget>
     <widget class="QSlider" name="R_Det_Slider">
      <property name="geometry">
       <rect>
        <x>110</x>
        <y>110</y>
        <width>160</width>
        <height>22</height>
       </rect>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
     <widget class="QLabel" name="label_12">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>50</y>
        <width>47</width>
        <height>13</height>
       </rect>
      </property>
      <property name="text">
       <string>Speed</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_13">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>110</y>
        <width>101</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Detection radius</string>
      </property>
     </widget>
     <widget class="QSpinBox" name="R_Count_Box">
      <property name="geometry">
       <rect>
        <x>110</x>
        <y>20</y>
        <width>51</width>
        <height>22</height>
       </rect>
      </property>
      <property name="maximum">
       <number>900</number>
      </property>
     </widget>
     <widget class="QSlider" name="R_Sep_Slider">
      <property name="geometry">
       <rect>
        <x>110</x>
        <y>80</y>
        <width>160</width>
        <height>22</height>
       </rect>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
     <widget class="QSlider" name="R_Speed_Slider">
      <property name="geometry">
       <rect>
        <x>110</x>
        <y>50</y>
        <width>160</width>
        <height>22</height>
       </rect>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
     <widget class="QLabel" name="label_14">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>140</y>
        <width>101</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Hunger</string>
      </property>
     </widget>
     <widget class="QSlider" name="R_Hunger_Slider">
      <property name="geometry">
       <rect>
        <x>110</x>
        <y>140</y>
        <width>160</width>
        <height>22</height>
       </rect>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
     <widget class="QLabel" name="R_Speed_Value">
      <property name="geometry">
       <rect>
        <x>280</x>
        <y>50</y>
        <width>31</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>0</string>
      </property>
     </widget>
     <widget class="QLabel" name="R_Det_Value">
      <property name="geometry">
       <rect>
        <x>280</x>
        <y>110</y>
        <width>31</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>0</string>
      </property>
     </widget>
     <widget class="QLabel" name="R_Sep_Value">
      <property name="geometry">
       <rect>
        <x>280</x>
        <y>80</y>
        <width>31</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>0</string>
      </property>
     </widget>
     <widget class="QLabel" name="R_Hunger_Value">
      <property name="geometry">
       <rect>
        <x>280</x>
        <y>140</y>
        <width>31</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>0</string>
      </property>
     </widget>
    </widget>
    <widget class="QGroupBox" name="Blue_Group">
     <property name="geometry">
      <rect>
       <x>340</x>
       <y>10</y>
       <width>321</width>
       <height>151</height>
      </rect>
     </property>
     <property name="title">
      <string>Blue birds</string>
     </property>
     <widget class="QLabel" name="label_4">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>80</y>
        <width>101</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Separation radius</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>20</y>
        <width>47</width>
        <height>13</height>
       </rect>
      </property>
      <property name="text">
       <string>No. birds</string>
      </property>
     </widget>
     <widget class="QSlider" name="B_Det_Slider">
      <property name="geometry">
       <rect>
        <x>110</x>
        <y>110</y>
        <width>160</width>
        <height>22</height>
       </rect>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>50</y>
        <width>47</width>
        <height>13</height>
       </rect>
      </property>
      <property name="text">
       <string>Speed</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_5">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>110</y>
        <width>101</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Detection radius</string>
      </property>
     </widget>
     <widget class="QSpinBox" name="B_Count_Box">
      <property name="geometry">
       <rect>
        <x>110</x>
        <y>20</y>
        <width>51</width>
        <height>22</height>
       </rect>
      </property>
      <property name="maximum">
       <number>900</number>
      </property>
     </widget>
     <widget class="QSlider" name="B_Sep_Slider">
      <property name="geometry">
       <rect>
        <x>110</x>
        <y>80</y>
        <width>160</width>
        <height>22</height>
       </rect>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
     <widget class="QSlider" name="B_Speed_Slider">
      <property name="geometry">
       <rect>
        <x>110</x>
        <y>50</y>
        <width>160</width>
        <height>22</height>
       </rect>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
     <widget class="QLabel" name="B_Det_Value">
      <property name="geometry">
       <rect>
        <x>280</x>
        <y>110</y>
        <width>31</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>0</string>
      </property>
     </widget>
     <widget class="QLabel" name="B_Sep_Value">
      <property name="geometry">
       <rect>
        <x>280</x>
        <y>80</y>
        <width>31</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>0</string>
      </property>
     </widget>
     <widget class="QLabel" name="B_Speed_Value">
      <property name="geometry">
       <rect>
        <x>280</x>
        <y>50</y>
        <width>31</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>0</string>
      </property>
     </widget>
    </widget>
    <widget class="QGroupBox" name="Blue_Group_2">
     <property name="geometry">
      <rect>
       <x>340</x>
       <y>160</y>
       <width>321</width>
       <height>81</height>
      </rect>
     </property>
     <property name="title">
      <string>Obstacles</string>
     </property>
     <widget class="QLabel" name="label_17">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>20</y>
        <width>81</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>No. obstacles</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_18">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>50</y>
        <width>47</width>
        <height>13</height>
       </rect>
      </property>
      <property name="text">
       <string>Radius</string>
      </property>
     </widget>
     <widget class="QSpinBox" name="Obs_Count_Box">
      <property name="geometry">
       <rect>
        <x>110</x>
        <y>20</y>
        <width>51</width>
        <height>22</height>
       </rect>
      </property>
      <property name="maximum">
       <number>20</number>
      </property>
     </widget>
     <widget class="QSlider" name="Obs_Radius_Slider">
      <property name="geometry">
       <rect>
        <x>110</x>
        <y>50</y>
        <width>160</width>
        <height>22</height>
       </rect>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
     <widget class="QLabel" name="Obs_Radius_Value">
      <property name="geometry">
       <rect>
        <x>280</x>
        <y>50</y>
        <width>31</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>0</string>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QWidget" name="Advanced_Tab">
    <attribute name="title">
     <string>Advanced Settings</string>
    </attribute>
    <widget class="QGroupBox" name="groupBox_3">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>321</width>
       <height>171</height>
      </rect>
     </property>
     <property name="title">
      <string>Green Birds</string>
     </property>
     <widget class="QGroupBox" name="groupBox_2">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>20</y>
        <width>301</width>
        <height>141</height>
       </rect>
      </property>
      <property name="autoFillBackground">
       <bool>true</bool>
      </property>
      <property name="title">
       <string>Force relative strengths</string>
      </property>
      <property name="flat">
       <bool>false</bool>
      </property>
      <widget class="QLabel" name="label_22">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>50</y>
         <width>111</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Alignment</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_21">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>20</y>
         <width>111</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Cohesion</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_23">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>80</y>
         <width>111</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Separation</string>
       </property>
      </widget>
      <widget class="QSlider" name="G_Coh_Strength_Slider">
       <property name="geometry">
        <rect>
         <x>100</x>
         <y>20</y>
         <width>160</width>
         <height>22</height>
        </rect>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
      <widget class="QSlider" name="G_Ali_Strength_Slider">
       <property name="geometry">
        <rect>
         <x>100</x>
         <y>50</y>
         <width>160</width>
         <height>22</height>
        </rect>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
      <widget class="QSlider" name="G_Sep_Strength_Slider">
       <property name="geometry">
        <rect>
         <x>100</x>
         <y>80</y>
         <width>160</width>
         <height>22</height>
        </rect>
       </property>
       <property name="maximum">
        <number>100</number>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
      <widget class="QLabel" name="label_24">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>110</y>
         <width>121</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Avoid Predators</string>
       </property>
      </widget>
      <widget class="QSlider" name="G_AvoidPred_Strength_Slider">
       <property name="geometry">
        <rect>
         <x>100</x>
         <y>110</y>
         <width>160</width>
         <height>22</height>
        </rect>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
      <widget class="QLabel" name="G_Coh_Strength_Value">
       <property name="geometry">
        <rect>
         <x>270</x>
         <y>20</y>
         <width>16</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>0</string>
       </property>
      </widget>
      <widget class="QLabel" name="G_Ali_Strength_Value">
       <property name="geometry">
        <rect>
         <x>270</x>
         <y>50</y>
         <width>16</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>0</string>
       </property>
      </widget>
      <widget class="QLabel" name="G_Sep_Strength_Value">
       <property name="geometry">
        <rect>
         <x>270</x>
         <y>80</y>
         <width>16</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>0</string>
       </property>
      </widget>
      <widget class="QLabel" name="G_AvoidPred_Strength_Value">
       <property name="geometry">
        <rect>
         <x>270</x>
         <y>110</y>
         <width>16</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>0</string>
       </property>
      </widget>
     </widget>
    </widget>
    <widget class="QGroupBox" name="groupBox_4">
     <property name="geometry">
      <rect>
       <x>340</x>
       <y>10</y>
       <width>321</width>
       <height>171</height>
      </rect>
     </property>
     <property name="title">
      <string>Blue Birds</string>
     </property>
     <widget class="QGroupBox" name="groupBox_5">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>20</y>
        <width>301</width>
        <height>141</height>
       </rect>
      </property>
      <property name="autoFillBackground">
       <bool>true</bool>
      </property>
      <property name="title">
       <string>Force relative strengths</string>
      </property>
      <property name="flat">
       <bool>false</bool>
      </property>
      <widget class="QLabel" name="label_31">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>50</y>
         <width>111</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Alignment</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_32">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>20</y>
         <width>111</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Cohesion</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_33">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>80</y>
         <width>111</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Separation</string>
       </property>
      </widget>
      <widget class="QSlider" name="B_Coh_Strength_Slider">
       <property name="geometry">
        <rect>
         <x>100</x>
         <y>20</y>
         <width>160</width>
         <height>22</height>
        </rect>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
      <widget class="QSlider" name="B_Ali_Strength_Slider">
       <property name="geometry">
        <rect>
         <x>100</x>
         <y>50</y>
         <width>160</width>
         <height>22</height>
        </rect>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
      <widget class="QSlider" name="B_Sep_Strength_Slider">
       <property name="geometry">
        <rect>
         <x>100</x>
         <y>80</y>
         <width>160</width>
         <height>22</height>
        </rect>
       </property>
       <property name="maximum">
        <number>100</number>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
      <widget class="QLabel" name="label_34">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>110</y>
         <width>121</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Avoid Predators</string>
       </property>
      </widget>
      <widget class="QLabel" name="B_Coh_Strength_Value">
       <property name="geometry">
        <rect>
         <x>270</x>
         <y>20</y>
         <width>16</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>0</string>
       </property>
      </widget>
      <widget class="QLabel" name="B_Ali_Strength_Value">
       <property name="geometry">
        <rect>
         <x>270</x>
         <y>50</y>
         <width>16</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>0</string>
       </property>
      </widget>
      <widget class="QLabel" name="B_Sep_Strength_Value">
       <property name="geometry">
        <rect>
         <x>270</x>
         <y>80</y>
         <width>16</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>0</string>
       </property>
      </widget>
      <widget class="QSlider" name="B_AvoidPred_Strength_Slider">
       <property name="geometry">
        <rect>
         <x>100</x>
         <y>110</y>
         <width>160</width>
         <height>22</height>
        </rect>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
      <widget class="QLabel" name="B_AvoidPred_Strength_Value">
       <property name="geometry">
        <rect>
         <x>270</x>
         <y>110</y>
         <width>16</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>0</string>
       </property>
      </widget>
     </widget>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
